### PR TITLE
Completed homework 1and2

### PR DESCRIPTION
**Pull request by bsaranga**
Please review and merge my first homework completion :)

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,39 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int numPeople = 2;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:9]: Use const modifier.

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,39 @@ class Program
...
+
+                BMIPrinter(weight, height);
+
+                numPeople--;
+            }
+
+            Console.ReadKey();
+        }
+
+        static void BMIPrinter(double weight, double height)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Instead of BMI name Bmi. Pascal case notation.

Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: This actually doesn't just print Bmi, it also calculates it.
I would say it's better to split it into two functions (maybe just one)
But don't couple printing with calculating- the two are very different things.

**Review by Almantask — CHANGES_REQUESTED** [review]
All looks great except the tiny nitpick of coupling.

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (bsaranga) [reply Src/BootCamp.Chapter/Program.cs]: Agreed.

Comment (bsaranga) [reply Src/BootCamp.Chapter/Program.cs]: Noted

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (bsaranga) [reply Src/BootCamp.Chapter/Program.cs:9]: When I use the const modifier I cannot decrement it in the while loop.

**Comment by bsaranga** [comment]
Requested changes have been made

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs:9]: Which is fine- you don't need to decrement numPeople in the loop. You need to increment the index until it reaches numPeople constant.

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**